app/controllers/candidate_interface/degrees/degrees_base_controller.rb

```diff
@@ -0,0 +1,11 @@
+module CandidateInterface
+  module Degrees
+    class DegreesBaseController < CandidateInterfaceController
```

Comment:
Namespaced, so can just call it `BaseController`. Will then be consistent with other example in the codebase.